src/Functional.CQS.AOP.IoC.SimpleInjector/Functional.CQS.AOP.IoC.SimpleInjector.xml

```diff
@@ -0,0 +1,222 @@
+<?xml version="1.0"?>
+<doc>
+    <assembly>
+        <name>Functional.CQS.AOP.IoC.SimpleInjector</name>
```

Comment:
Consider:
- Adding these generated-on-build XML files to your `.gitignore`.
- Manually removing the file from source control. 
  - According to https://stackoverflow.com/questions/2047465/how-can-i-delete-a-file-from-git-repo#2047477 and https://git-scm.com/docs/git-rm, you can use `git rm --cached PathToFile` to achieve this.
  - The `--cached` option ensures you just remove it from source control, as opposed to physically deleting the file from your computer.